### ISSUE TITLE
new stooq datareader no longer downloads indices

### ISSUE THREAD

**Issue by EcoFin** [pydata/pandas-datareader#594]
The stooq 0.7.0 datareader downloads individual symbols provided as either "AAPL" or "AAPL.US".  But it will no longer download indices (e.g. "^SPX"), and returns an empty dataframe.  Apparently, stooq.py was rewritten to automatically append ".US" in the absence of any other country indicator.  But indices do not take a country indicator (it's "^SPX", not "^SPX.US").  For now, I have simply replaced the new stooq.py with the version 0.6.0 one.  But a fix would be welcome.

**Comment by gliptak** [comment]
This was addressed with https://github.com/pydata/pandas-datareader/pull/591

**Comment by Rias9371J** [comment]
@gliptak I just tried this; but it still doesn't work 
```
import pandas_datareader.data as web
print(web.DataReader('^DJI', 'stooq'))
```

**Comment by gliptak** [comment]
@Rias9371J confirmed, #591 didn't seem to have been released

@bashtage ```0.7``` releases at

https://github.com/pydata/pandas-datareader/releases

do not seem to have current version of https://github.com/pydata/pandas-datareader/blob/master/pandas_datareader/stooq.py

Are the ```0.7``` releases patch only?

**Comment by addisonlynch** [comment]
@gliptak @Rias9371J correct...0.7.x releases were patches for py2 compatibility. The work in #591 will be released in 0.8.0. For now installing latest dev version via master git branch should solve the issue.

**Comment by Rias9371J** [comment]
@addisonlynch Awesome! Thank you - will stay tuned;
